packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -443,6 +448,13 @@ export interface DDSFuzzSuiteOptions {
 	 * test case. See {@link MinimizationTransform} for additional context.
 	 */
 	skipMinimization?: boolean;
+
+	/**
+	 * An optional IdCompressor that can will be passed to the constructed MockDataStoreRuntime instance.
```

Comment:
nit: This is probably my typo, but now I'm noticing it. Should say `An optional IdCompressor that will be passed to the constructed MockDataStoreRuntime instance`